src/modules/feature-modules/innovator/services/innovator.service.ts

```diff
@@ -16,6 +16,20 @@ export type GetInnovationTransfersDTO = {
   innovation: { id: string, name: string, owner: string };
 }[];
 
+export type GetInnovationCollaboratorInvitesDTO = {
+  id: string;
+  innovation: { id: string, name: string, owner: string; description: string };
+}[];
+
+export enum InnovationCollaboratorStatusEnum {
```

Comment:
@kduarte-bjss I think this should be at `innovation.enums.ts`

Reply:
Done